lib/clubs.jsx

```diff
@@ -0,0 +1,116 @@
+
+var Page = require('./page.jsx');
+var HeroUnit = require('./hero-unit.jsx');
+var Blockquote = require('./blockquote.jsx');
+var IconLink = require('./icon-link.jsx');
+var PageEndCTA = require('./page-end-cta.jsx');
+
+var WebLitMap = React.createClass({
+  render: function() {
+    return(
+      <div className="row">
+        <div className="col-sm-12 col-md-12 col-lg-12">
+          <h2>The Global Web Literacy Movement</h2>
+          <div>placeholder for map</div>
+        </div>
+      </div>
+    );
+  }
+});
+
+var HowClubWorks = React.createClass({
+  render: function() {
+    return(
+      <div className="row">
+        <div className="col-sm-4 col-md-4 col-lg-4">
+          <img src="/img/icon-how-do-clubs-work.svg" alt="icon how do clubs work" />
+        </div>
+        <div className="col-sm-8 col-md-8 col-lg-8">
+          <h2>How do Clubs work?</h2>
+          <ul>
+            <li>Grow the web literary of leaners</li>
+            <li>Meet regularly in classrooms, libraries, coffee shops &mdash; anywhere!</li>
+            <li>Teach with open practices</li>
+            <li>Guide people to learn by making</li>
+            <li>Connect with local and global networks</li>
+          </ul>
+        </div>
+      </div>
+    );
+  }
+});
+
+var IconLinks = React.createClass({
+  ICON_LINKS: [
+    {
+      linkTo: "/fixme",
+      imgSrc: "/img/icon-curriculum.svg",
+      imgAlt: "icon curriculum",
+      head: "Curriculum",
+      subhead: "Modular Web Literacy curriculum"
+    },
+    {
+      linkTo: "/fixme",
+      imgSrc: "/img/icon-connect.svg",
+      imgAlt: "icon connect",
+      head: "Connect",
+      subhead: "Connect with other Club Leaders"
+    },
+    {
+      linkTo: "/fixme",
+      imgSrc: "/img/icon-tips.svg",
+      imgAlt: "icon tips",
+      head: "Helpful Tips",
+      subhead: "Tips for running your Club"
+    }
+  ],
+  render: function() {
+    return(
+      <div className="row">
+        <div className="col-sm-4 col-md-4 col-lg-4">
+          <IconLink info={this.ICON_LINKS[0]} />
+        </div>
+        <div className="col-sm-4 col-md-4 col-lg-4">
+          <IconLink info={this.ICON_LINKS[1]} />
+        </div>
+        <div className="col-sm-4 col-md-4 col-lg-4">
+          <IconLink info={this.ICON_LINKS[2]} />
+        </div>
+      </div>
+    );
+  }
+});
+
+
+var Clubs = React.createClass({
+  statics: {
+    pageClassName: "clubs"
+  },
+  render: function() {
+    var ctaLink = "/fixme-cta";
+    return (
+      <div>
+        <HeroUnit image="/img/hero-clubs.jpg">
+          <h1>Mozilla Learning Clubs</h1>
+          <div><Link to={ctaLink} className="btn btn-awsm">Add Your Club</Link></div>
+          <div><p>or <Link to="/fixme">find out more</Link> about us</p></div>
+        </HeroUnit>
+        <WebLitMap/>
+        <HowClubWorks/>
+        <Blockquote className="primary-quote"
+                    imgSrc="/img/mikko-finland@2x.png" imgAlt="Mikko Finland Quote"
```

Comment:
Oh yea, sorry was gonna raise this question as I'm not sure either.  I know for `background-image` we can simply use CSS to detect and use the right version of file.  Not sure if there's something for `<img>` as well.  Maybe @gesa would know?